### PR TITLE
feat: [CDS-38862]: Re-Design Active Deployments tab

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
#### Screenshots

https://user-images.githubusercontent.com/98325173/178987546-01e126bc-55d4-4d33-b18b-0214d37d799a.mov

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by Sagar-Rajput7** [comment]
retrigger titlecheck

**Review by bhavya-sinha — APPROVED** [review]

File: src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancesContentV2.tsx [hunk 1]
```diff
@@ -130,21 +130,26 @@ const getSummaryTableData = (instanceGroupedByArtifact?: InstanceGroupedByArtifa
...
-const RenderArtifactVersion: Renderer<CellProps<TableRowData>> = ({
+export const RenderArtifactVersion: Renderer<CellProps<TableRowData>> = ({
   row: {
     original: { artifactVersion, showArtifact }
   }
 }) => {
   return showArtifact ? (
-    <Text font={{ size: 'small', weight: 'semi-bold' }} lineClamp={1} color={Color.GREY_800}>
+    <Text
+      style={{ maxWidth: '200px', paddingRight: 'var(--spacing-5)' }}
```
Comment (depanshu-dhiman) [inline src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancesContentV2.tsx:140]: Use classname

File: src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancesContentV2.tsx [hunk 2]
```diff
@@ -174,15 +179,20 @@ const RenderEnvironment: Renderer<CellProps<TableRowData>> = ({
...
     original: { infraName, totalInfras }
   }
 }) => {
   return infraName ? (
     <Container flex>
-      <Layout.Horizontal>
-        <Text font={{ size: 'small', weight: 'semi-bold' }} lineClamp={1} color={Color.GREY_800}>
+      <Layout.Horizontal flex={{ alignItems: 'center', justifyContent: 'flex-start' }} width={'100%'}>
+        <Text
+          style={{ paddingRight: 'var(--spacing-2)' }}
```
Comment (depanshu-dhiman) [inline src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancesContentV2.tsx:191]: Use classname

**Review by depanshu-dhiman — APPROVED** [review]
Very minor comments can be addressed in next PR

**Review by bhavya-sinha — APPROVED** [review]

**Pull request merged**